### PR TITLE
increase models version to 6.0.0 + create new packages

### PR DESCRIPTION
**Pull request by priesgo**
That's it 6.0.0 the version in development.
New package versions for reports and CVA.

### CODE REVIEW

**Comment by priesgo** [comment]
please @Antonior26 @greglever @gerlv @jancellor could you review?

File: schemas/IDLs/org.gel.models.report.avro/5.0.0-SNAPSHOT/ReportVersionControl.avdl [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+/**
+This protocol defines the Participants and Pedigree
+*/
+protocol VersionControl {
+
+    record ReportVersionControl {
+
+        /** This is the version for the entire set of data models as referred to the Git release tag
+        */
+        string gitVersionControl = "4.2.0";
```
Comment (greglever) [inline schemas/IDLs/org.gel.models.report.avro/5.0.0-SNAPSHOT/ReportVersionControl.avdl]: should this be `string gitVersionControl = "5.0.0";` ?

**Comment by greglever** [comment]
@priesgo just a small comment, otherwise the tests are passing now:
```
INFO:root:Build/s finished succesfully!
 ---> c87cca140684
Removing intermediate container b7303336c854
Successfully built c87cca140684
Successfully tagged gel:latest
root@727013bd9620:/gel# cd GelReportModels/
root@727013bd9620:/gel/GelReportModels# source ./.env/bin/activate
(.env) root@727013bd9620:/gel/GelReportModels# python -m unittest discover
............................
----------------------------------------------------------------------
Ran 28 tests in 2.809s

OK
(.env) root@727013bd9620:/gel/GelReportModels# 
```

**Comment by greglever** [comment]
Also:
```
INFO:root:Build/s finished succesfully!
 ---> f848be2c585f
Removing intermediate container 9288fdf8914c
Successfully built f848be2c585f
Successfully tagged gel:latest
```
the docs and java part of the build works fine as well.

File: schemas/IDLs/org.gel.models.report.avro/5.0.0-SNAPSHOT/ReportVersionControl.avdl (hunk 1 above)
Comment (priesgo) [reply schemas/IDLs/org.gel.models.report.avro/5.0.0-SNAPSHOT/ReportVersionControl.avdl]: agree

**Pull request merged**